tests/unit/pipeline/test_ner_re_pipeline.py

```diff
@@ -0,0 +1,390 @@
+import dataclasses
+
+import pytest
+from pie_modules.annotations import BinaryRelation, LabeledSpan
+from pie_modules.documents import TextBasedDocument
+from pytorch_ie import AnnotationLayer
+from pytorch_ie.core import annotation_field
+from pytorch_ie.models import (
+    TransformerSpanClassificationModel,
+    TransformerTextClassificationModel,
+)
+from pytorch_ie.taskmodules import (
+    TransformerRETextClassificationTaskModule,
+    TransformerSpanClassificationTaskModule,
+)
```

Comment:
please add a comment why these imports are necessary

Reply:
and maybe import them locally in the `test_ner_re_pipeline` method instead of here to have it at the place where it is required?